thorntail and payara _microprofile_tck build failure jdk11

For both [hs](https://ci.adoptopenjdk.net/view/Test_external/job/openjdk11_hs_externaltest_extended_x86-64_linux/115) and [j9](https://ci.adoptopenjdk.net/view/Test_external/job/openjdk11_j9_externaltest_extended_x86-64_linux/104): 

payara_microprofile_tck
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.20.1:test (default-test) on project microprofile-openapi-tck-runner: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.20.1:test failed.: NullPointerException -> [Help 1]

for thorntail_microprofile_tck:
00:53:12 [ERROR] Failed to execute goal on project testsuite-microprofile-jwt: Could not resolve dependencies for project io.thorntail.testsuite:testsuite-microprofile-jwt:jar:2.3.1.Final-SNAPSHOT: The following artifacts could not be resolved: io.thorntail:microprofile-jwt:jar:2.3.1.Final-SNAPSHOT, io.thorntail:jaxrs-jsonp:jar:2.3.1.Final-SNAPSHOT, io.thorntail:arquillian:jar:2.3.1.Final-SNAPSHOT: Could not find artifact io.thorntail:microprofile-jwt:jar:2.3.1.Final-SNAPSHOT -> [Help 1]

--- a/thirdparty_containers/kafka/playlist.xml
+++ b/thirdparty_containers/kafka/playlist.xml
@@ -23,5 +23,6 @@
 		<groups>
 			<group>external</group>
 		</groups>
+		<disabled>https://github.com/AdoptOpenJDK/openjdk-tests/issues/855</disabled>
 	</test>
 </playlist>

--- a/thirdparty_containers/payara-mp-tck/playlist.xml
+++ b/thirdparty_containers/payara-mp-tck/playlist.xml
@@ -23,6 +23,9 @@
 			 docker cp payara-mp-tck:/MicroProfile-TCK-Runners/MicroProfile-OpenAPI/tck-runner/target/surefire-reports/junitreports $(REPORTDIR)/external_test_reports; \
 			 docker rm payara-mp-tck
 		</command>
+		<subsets>
+			<subset>8</subset>
+		</subsets>
 		<levels>
 			<level>extended</level>
 		</levels>

--- a/thirdparty_containers/thorntail-mp-tck/playlist.xml
+++ b/thirdparty_containers/thorntail-mp-tck/playlist.xml
@@ -24,6 +24,9 @@
 			 docker cp thorntail-mp-tck:/thorntail/testsuite//testsuite-microprofile-jwt/target/surefire-reports $(REPORTDIR)/external_test_reports; \
 			 docker rm thorntail-mp-tck
 		</command>
+		<subsets>
+			<subset>8</subset>
+		</subsets>
 		<levels>
 			<level>extended</level>
 		</levels>

--- a/thirdparty_containers/wildfly/playlist.xml
+++ b/thirdparty_containers/wildfly/playlist.xml
@@ -23,5 +23,6 @@
 		<groups>
 			<group>external</group>
 		</groups>
+		<disabled>https://github.com/AdoptOpenJDK/openjdk-tests/issues/855</disabled>
 	</test>
 </playlist>

